draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -70,8 +70,17 @@ type WorkingUndefinedProps = {
 export type LabelProps = LabelPropsGeneric &
   (WorkingProps | WorkingUndefinedProps)
 
-export type IconButtonProps = GenericProps &
+type NonEmptyString<T extends string> = T extends "" ? never : T
```

Comment:
Do we have somewhere we put shared types? I imagine this would be helpful in other components too :)